espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.kt

```diff
@@ -34,6 +41,25 @@ import org.junit.Assert.assertEquals
  */
 @LargeTest
 class EspressoServerRunnerTest {
+
+    @get:Rule
+    val composeRule = AndroidComposeTestRule(
+        activityRule = EmptyTestRule(),
+        activityProvider = { error("Can't provide current activity") }
+    ).also {
+        composeTestRule = it
+    }
+
+    private val syncComposeClock = Thread{
+        while (!Server.isStopRequestReceived){
+            if (context.driverStrategy is ComposeDriver) {
+                composeTestRule.mainClock.advanceTimeByFrame()
+            }
+            // Let Android run measure, draw and in general any other async operations. AndroidComposeTestRule.android.kt:325
+            Thread.sleep(10)
```

Comment:
We could move the magic value to a cosntant and give it a proper self-descriptive name to make it easier to read

Reply:
done

Reply:
hm, are you sure you've pushed the changes? I don't observe any changes

Reply:
Sorry, I forgot to push. It should be there now